auraed/src/cells/cell_service/cells/cells_cache.rs

```diff
@@ -63,4 +63,6 @@ pub trait CellsCache {
 
     /// Sends a [SIGKILL] to all Cells, ignoring any errors.
     fn broadcast_kill(&mut self);
+
+    fn entries(&self) -> Vec<&Cell>;
```

Comment:
`entries` makes it seems like the [Entry api](https://doc.rust-lang.org/std/collections/hash_map/enum.Entry.html) will be exposed. I'd call it `values`.